src/app/App.tsx

```diff
@@ -0,0 +1,42 @@
+import { useState } from "react"
+import reactLogo from "../assets/react.svg"
+import "./App.css"
+import i18nManager from "./i18n"
+import Layout from "./providers/layout"
```

Comment:
Layout component is not really a provider. It is just a component so it should be stored in another place.

Reply:
Done